### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by tanjunlem**

### CODE REVIEW

**Comment by MasterODST** [comment]
Question what will this sub system coincide with in terms of the rest of the system?

**Comment by MasterODST** [comment]
Also on the gathering aspect of the customers information why not add a field to gather their email addresses as well? Since it would be easier to contact them via email if we ever wish to advertise some promotions.

**Comment by chunweng428** [comment]
The name for pull request seems like doesn't match with what you have pulled here. Can you check it again?

**Comment by tanjunlem** [comment]
Ok, I will check it

**Comment by tanjunlem** [comment]
I had did my research, I discovered that most of the customers like the company just send promotion to their SMS instead of email because they were not always check their email except SMS. They were afraid of missed the date during the promotion.

**Comment by tanjunlem** [comment]
I don't really understand your question, can you explain to me ?
 (Question what will this sub system coincide with in terms of the rest of the system?)

File: process/AddCustomer_Coding.txt [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+    private void btnAddActionPerformed(java.awt.event.ActionEvent evt) {                                       
+        Customer c = new Customer("", txtCustName.getText(), txtCustIC.getText(), txtCustPhoneNo.getText(), txtCustAddress.getText());
+        UJS.addCustomer(c);
+        this.dispose();
+    }                                      
+
+    public static void main(String args[]) {
+        /* Set the Nimbus look and feel */
+        //<editor-fold defaultstate="collapsed" desc=" Look and feel setting code (optional) ">
```
Comment (TeckYong) [inline process/AddCustomer_Coding.txt:111]: What does this line of code do

File: process/AddCustomer_Coding.txt [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+                    .addComponent(jLabel2)
+                    .addComponent(txtCustIC, javax.swing.GroupLayout.PREFERRED_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.PREFERRED_SIZE))
+                .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.UNRELATED)
+                .addGroup(layout.createParallelGroup(javax.swing.GroupLayout.Alignment.BASELINE)
+                    .addComponent(jLabel3)
+                    .addComponent(txtCustPhoneNo, javax.swing.GroupLayout.PREFERRED_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.PREFERRED_SIZE))
+                .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.UNRELATED)
+                .addGroup(layout.createParallelGroup(javax.swing.GroupLayout.Alignment.BASELINE)
+                    .addComponent(jLabel4)
+                    .addComponent(txtCustAddress, javax.swing.GroupLayout.PREFERRED_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.PREFERRED_SIZE))
```
Comment (TeckYong) [inline process/AddCustomer_Coding.txt:94]: follow the coding format.. align code properly

File: process/AddCustomer_Coding.txt [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+                .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.RELATED, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
```
Comment (TeckYong) [inline process/AddCustomer_Coding.txt:95]: same

File: process/AddCustomer_Coding.txt [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+                .addContainerGap())
+        );
+        layout.setVerticalGroup(
+            layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
+            .addGroup(layout.createSequentialGroup()
+                .addContainerGap()
+                .addGroup(layout.createParallelGroup(javax.swing.GroupLayout.Alignment.BASELINE)
+                    .addComponent(jLabel1)
+                    .addComponent(txtCustName, javax.swing.GroupLayout.PREFERRED_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.PREFERRED_SIZE))
+                .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.UNRELATED)
```
Comment (TeckYong) [inline process/AddCustomer_Coding.txt:83]: same

File: process/AddCustomer_Coding.txt (hunk 1 above)
Comment (chunweng428) [reply process/AddCustomer_Coding.txt:111]: There's no coding there, it's just some comments.

**Review by chunweng428 — APPROVED** [review]

**Review by TeckYong — APPROVED** [review]

**Pull request merged**